Dockerfile

```
@@ -23,13 +23,17 @@ RUN wget https://raw.githubusercontent.com/couchbase/sync_gateway/$COMMIT/bootst
     chmod +x bootstrap.sh && \
     ./bootstrap.sh -c $COMMIT -p sg
 
+ARG SG_EDITION=CE
+
 # Build the Sync Gateway binary
 RUN ./build.sh -v
 
 
 # Stage to run the SG binary from the previous stage
 FROM ubuntu:latest as runner
 
-COPY --from=builder /go/godeps/bin/sync_gateway .
+ARG SG_FILENAME=sync_gateway_ce
+
+COPY --from=builder /go/godeps/bin/$SG_FILENAME .
```

Lets copy the binary to a fixed location/filename, otherwise the entrypoint below will fail to run correctly.

```suggestion
COPY --from=builder /go/godeps/bin/$SG_FILENAME /sync_gateway
```

---

Yes! I just came back to make exactly this change.

The destination should maybe be `./sync_gateway` (relative), but I think WORKDIR is the root directory so that's only a theoretical issue.